composer.json

```
@@ -14,8 +14,8 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "squizlabs/php_codesniffer": "^3.6"
+        "php": ">=7.2",
+        "squizlabs/php_codesniffer": "4.0.*"
```

Let's be less strict in phpcs version constraint:
```suggestion
        "squizlabs/php_codesniffer": "^4.0"
```